CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 * `wandb-core` now supports Artifact file caching by @moredatarequired in https://github.com/wandb/wandb/pull/7364 and https://github.com/wandb/wandb/pull/7366
 * Added artifact_exists() and artifact_collection_exists() methods to Api to check if an artifact or collection exists by @amusipatla-wandb in https://github.com/wandb/wandb/pull/7483
 * `wandb launch -u <git-uri | local-path> ` creates and launches a job from the given source code by @bcsherma in https://github.com/wandb/wandb/pull/7485
+* Added save() method on ArtifactCollection to allow persisting changes by @amusipatla-wandb in https://github.com/wandb/wandb/pull/7555
```

Comment:
nit: you can shorten this to just `Added ArtifactCollection.save() by ...`

"allow persisting changes" is implied by the method name IMO